generic_scripts/nlk.py

```diff
@@ -0,0 +1,167 @@
+import numpy as _np
+import matplotlib.pyplot as _plt
+from mathphys.constants import vacuum_permeability
+from mathphys.beam_optics import beam_rigidity
+
+
+class Wire:
+    """."""
+    def __init__(self, position, current):
+        self.pos = position  # Column vector
+        self.curr = current
+
+    def Bfield(self, x, y):
```

Comment:
Use lower case here. Besides, since this is a method, use one of the prefixes: `get_`, `calc_`, or similar.
Maybe a good name would be: `calc_magnetic_field()`